script.js

```diff
@@ -172,24 +185,26 @@ async function getArticleForLocation() {
 
 async function getNearbyArticle() {
   console.info('Finding nearby article');
-  const response = await fetchWithTimeout(wikiUrl('action=query&format=json&origin=*&generator=geosearch&ggsradius=10000&ggsnamespace=0&ggslimit=50&formatversion=2&ggscoord=' + encodeURIComponent(position.coords.latitude) + '%7C' + encodeURIComponent(position.coords.longitude), true, true));
+  const latitude = position.coords.latitude.toFixed(4)
+  const longitude = position.coords.longitude.toFixed(4)
+  const response = await fetchWithTimeout(wikiUrl('action=query&format=json&origin=*&generator=geosearch&ggsradius=10000&ggsnamespace=0&ggslimit=50&formatversion=2&ggscoord=' + encodeURIComponent(latitude) + '%7C' + encodeURIComponent(longitude), true, true));
   if (!response.ok) {
```

Comment:
round up coordinate values to reliably cache the inconsistent high precision of geolocation results